src/class/NCSCharacter/server/functions/removeAccount.lua

```diff
@@ -0,0 +1,20 @@
+---removeAccount
+---@public
+---@param self NCSCharacter
+---@param accountName string
+---@param amount number
+function NCSCharacter.removeAccount(self, _, accountName, amount)
+    if (not self) then
+        return NCS:die("Character not found.")
+    end
+
+    if (not self.accounts[accountName]) then
+        return NCS:die("Account not found.")
+    end
+    
```

Comment:
```suggestion
    amount = (amount or 0)
```

Reply:
If i do that account can be negative

Reply:
no, unless you accidentally have no value for the amount variable, you won't get an error.

Reply:
Ah, yes i'm stupid